template/template_test.go

```diff
@@ -350,14 +350,22 @@ func checkCVEBody(t *testing.T, body string, custom map[string]interface{}) {
 	t.Helper()
 	assert.True(t, strings.Contains(body, toString(custom["repo_url"])))
 	assert.True(t, strings.Contains(body, toString(custom["scanned_at"])))
+	assert.True(t, strings.Contains(body, toString(custom["total_dependencies"])))
+	dirDepArr := toArrMap(custom["direct_updates"])
+	assert.NotNil(t, dirDepArr)
+	transDepArr := toArrMap(custom["transitive_updates"])
+	assert.NotNil(t, transDepArr)
```

Comment:
length assertion for `direct_updates` and `transitive_updates` is missing.

Reply:
@nurali-techie, for cve, we have two files, one is cve.basic and one is cve.many. I skipped this one because it would be different for both so a generalized test is not feasible here.

Reply:
yes, you are right.  so in that case remove len check from `checkVersionBody()` too as it make it not re-usable.  I overlooked this before and ask to add length change but now I think we can't add it here.

Reply:
@nurali-techie, Done. PTAL